Sources/TuistLoader/ProjectDescriptionHelpers/ProjectDescriptionHelpersBuilder.swift

```diff
@@ -176,12 +176,6 @@ public final class ProjectDescriptionHelpersBuilder: ProjectDescriptionHelpersBu
             return projectDescriptionHelpersModule
         }
 
-        // If the same helpers directory has been previously compiled
-        // we delete it before compiling the new changes.
-        if FileHandler.shared.exists(helpersCachePath) {
-            try await fileSystem.remove(helpersCachePath)
-        }
```

Comment:
Should we still delete the helpers directory but instead delete only the `helpersModuleCachePath` directory?

Reply:
How would you know that path? It is based on a hash we dont posses for this run.

Reply:
Not that the helpersModuleCachePath is guaranteed to not be present (checking it in the line above)

Reply:
Im guessing you want to clear other cached builds? If so how would you distinguish those? You don’t have the other hashes. Im still not convinced we should delete any cache here. Let the user decide when to empty it with tuist clean

Reply:
ahh, you are right, my bad. Then I'm fully onboard with this change 👍